api/database/helpers/ApiEnums.php

```diff
@@ -387,6 +387,7 @@ public static function armedForcesStatuses(): array
     /**
      * Publishing Groups
      */
+    const PUBLISHING_GROUP_IAP = 'IAP';
```

Comment:
Could you update the seeder for the IAP to use this?
https://github.com/GCTC-NTGC/gc-digital-talent/blob/935f1aa95af8b8b333ec3f8113899ccbbf67c6ac/api/database/seeders/PoolSeeder.php#L45

Reply:
Thanks for pointing that out, I didn't realize it existed 😜 Updated in [update IAP pool seeder](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6521/commits/0507690c1ef835ea8e3a43cfac06a51b3e0a97f6)